web/templates/copy_message_button.hbs

```diff
@@ -1,3 +1,5 @@
 <div class="copy_button_base copy_message tippy-zulip-tooltip" data-tippy-content="{{t 'Copy and close' }}" aria-label="{{t 'Copy and close' }}" role="button">
-    {{> copy_to_clipboard_svg }}
```

Comment:
What in the world is going on with these templates files? I don't understand why you're duplicating things here, or why you're copying and pasting the contents of `copy_to_clipboard.svg`

Reply:
This was something I was experimenting with and got into the pr by mistake, I've fixed it now.